shoov/themes/custom/template_holder/template_holder.info

```diff
@@ -0,0 +1,6 @@
+name = Template Holder
+description = This theme is not to be activated.
```

Comment:
Change the description, something like
"The theme to hold the mimemail template."